module/VuFind/src/VuFind/Db/Service/CommentsService.php

```diff
@@ -165,4 +165,45 @@ public function changeResourceId(int $old, int $new): void
     {
         $this->getDbTable('comments')->update(['resource_id' => $new], ['resource_id' => $old]);
     }
+
+    /**
+     * Get a paginated result of all comments by user id
```

Comment:
Add punctuation for consistency.

```suggestion
     * Get a paginated result of all comments by user id.
```